prow/secretutil/censor_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package secretutil
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func copyBytes(input []byte) []byte {
+	output := make([]byte, len(input))
+	copy(output, input)
+	return output
+}
+
+var text = []byte("secret SECRET c2VjcmV0 sEcReT")
```

Comment:
You can replace this and copyBytes with a `func() []byte`. Lets scope this to the testcase where its actually used though to not make it pop up in IDE autocompletion